.github/how_to_compile_gudhi_in_a_conda_env.md

```diff
@@ -0,0 +1,97 @@
+# Install a conda development environment to compile GUDHI
+
+## Install miniconda
+
+Download the [installer](https://docs.conda.io/en/latest/miniconda.html) required by your system and follow the [instructions](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+## Create a dedicated environment
+
+```bash
+conda install -c conda-forge mamba  # installation with mamba is faster
+conda create --name gudhi
+conda activate gudhi
+mamba install -c conda-forge python cmake doxygen eigen cgal-cpp
+```
+
+Some of the requirements are in the gudhi repository (please refer to
+[how to use github to contribute to gudhi](how_to_use_github_to_contribute_to_gudhi.md)).
+In the gudhi repository - let's call it `/workdir/gudhi` i.e. - once submodules are initialised:
+
+```bash
+pip install -r ext/gudhi-deploy/build-requirements.txt 
+pip install -r ext/gudhi-deploy/test-requirements.txt  # pytorch can be painful to install - not mandatory
+```
+
+## Compilation
+
+In order to compile all c++ utilities, examples, benchmarks, unitary tests, and python module:
+```bash
+cd /workdir/gudhi
+rm -rf build
+mkdir build
+cd build
+# To build all even examples and benchmarks
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=$CONDA_PREFIX -DWITH_GUDHI_EXAMPLE=ON -DWITH_GUDHI_BENCHMARK=ON ..
+```
+
+### Specific python compilation
+
+In order to compile only python module
+```bash
+cd /workdir/gudhi
+rm -rf build
+mkdir build
+cd build
+# To build all even examples and benchmarks
```

Comment:
Is this comment ok for python?

Reply:
No, python examples are not built. Do you want me to precise ?

Reply:
If the comment says something that doesn't match reality, then I think the comment needs to be changed (or removed). It was probably just a copy-paste that was right elsewhere but not in this location.

Reply:
Ok, I got it, sorry... Yes it was a bad copy/paste. Done on b440e2b6